certif/app/templates/authenticated/sessions/update.hbs

```diff
@@ -19,12 +19,13 @@
         maxlength="255"
         class="input"
         @value={{this.session.address}}
+        aria-required="true"
       />
-      {{#each this.session.errors.address as |error|}}
+      {{#if this.isSessionAddressMissing}}
         <div class="session-form__error error-message">
-          {{error.message}}
+          {{t "pages.sessions.new.errors.SESSION_ADDRESS_REQUIRED"}}
         </div>
-      {{/each}}
+      {{/if}}
```

Comment:
**Remarque (a11Y) :**  
Il faudrait que le message d'erreur soit relié au champ par un id.

<input
aria-describedby="tutu-id"
/>
<p id="tutu-id">
     {{#if this.isSessionAddressMissing}}
        message d'erreur
      {{/if}}
</p>